src/components/composites/Skeleton/SkeletonCircle.tsx

```diff
@@ -3,10 +3,10 @@ import { usePropsResolution } from '../../../hooks';
 import type { ISkeletonProps } from './types';
 import Skeleton from './Skeleton';
 
-const SkeletonCircle = ({ children, ...props }: ISkeletonProps) => {
+const SkeletonCircle = ({ children, ...props }: ISkeletonProps, ref: any) => {
```

Comment:
> This does not appear to be correct, for one, it shouldn't pass the value into the FC like this.

It should.

> However, if you follow the other components they appear to have it but it should be ref?

You mean optional? Because ForwardRefRenderFunction requires a ref, or null, not an optional ref.

Reply:
```ts
function forwardRef<T, P = {}>(render: ForwardRefRenderFunction<T, P>): ForwardRefExoticComponent<PropsWithoutRef<P> & RefAttributes<T>>;
```

and

```ts
type ForwardedRef<T> = ((instance: T | null) => void) | MutableRefObject<T | null> | null;

    interface ForwardRefRenderFunction<T, P = {}> {
        (props: PropsWithChildren<P>, ref: ForwardedRef<T>): ReactElement | null;
        displayName?: string;
        // explicit rejected with `never` required due to
        // https://github.com/microsoft/TypeScript/issues/36826
        /**
         * defaultProps are not supported on render functions
         */
        defaultProps?: never;
        /**
         * propTypes are not supported on render functions
         */
        propTypes?: never;
    }
```

Reply:
> You mean optional? Because ForwardRefRenderFunction requires a ref, or null, not an optional ref.

If we're following consistency with other components they use `ref?: any` though that to me is incorrect anyway since it bypasses Typescript checking.

Reply:
Sorry I know it was merged, I didn't know I had to press "Submit Review" for it to show.

Reply:
Some [other components](https://github.com/GeekyAnts/NativeBase/blob/master/src/components/composites/AppBar/AppBar.tsx) also use `ref: any`. It's not consistent, sure, but it's not wrong. Straightening out this inconsistency could be another PR, but brace yourself because the generics can alternate between mobile and web depending on the components you're talking about.